ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -46,8 +46,15 @@ for f in $files
 do
   SECRET_NAME=$(yq -r .spec.secretRef.name "/tmp/credentials-request/${f}")
   SECRET_NAMESPACE=$(yq -r .spec.secretRef.namespace "/tmp/credentials-request/${f}")
-  filename=manifest_${SECRET_NAMESPACE}_secret.yml
-  cat >> "${SHARED_DIR}/${filename}" << EOF
+  FEATURE_GATE=$(yq -r '.metadata.annotations."release.openshift.io/feature-gate"' "/tmp/credentials-request/${f}")
+
+# 4.10 includes techpreview of CAPI which without the namespace: openshift-cluster-api
+# fails to bootstrap. Below checks if TechPreviewNoUpgrade is annotated and if so skips
+# creating that secret.
+
+  if [[ $FEATURE_GATE != *"TechPreviewNoUpgrade"* ]]; then
```

Comment:
With the indentation of the subsequent cat, I think it would be a bit easier to read if we were to use a `continue`.

```suggestion
  if [[ $FEATURE_GATE == *"TechPreviewNoUpgrade"* ]]; then
    continue
  fi
```

Reply:
ack